certbot/certbot/_internal/main.py

```diff
@@ -921,6 +922,62 @@ def update_account(config: configuration.NamespaceConfig,
     return None
 
 
+def fetch_account(config: configuration.NamespaceConfig,
+                   unused_plugins: plugins_disco.PluginsRegistry) -> Optional[str]:
+    """Fetch account info on the server.
+
+    :param config: Configuration object
+    :type config: configuration.NamespaceConfig
+
+    :param unused_plugins: List of plugins (deprecated)
+    :type unused_plugins: plugins_disco.PluginsRegistry
+
+    :returns: `None`
+    :rtype: None
+
+    """
+    # Portion of _determine_account logic to see whether accounts already
+    # exist or not.
+    account_storage = account.AccountFileStorage(config)
+    accounts = account_storage.find_all()
+
+    if not accounts:
+        return "Could not find an existing account to fetch."
+
+    acc, acme = _determine_account(config)
+    cb_client = client.Client(config, acc, None, None, acme=acme)
+
+    if not cb_client.acme:
+        raise errors.Error("ACME client is not set.")
+
+    regr = cb_client.acme.query_registration(acc.regr)
+
+    if config.verbose_count > 0:
+        display_util.notify(f"Account URI: {regr.uri}")
+
+        thumbprint = jose.b64encode(regr.body.key.thumbprint(
+            hash_function=SHA256)).decode()
+        display_util.notify(f"Account thumbprint: {thumbprint}")
```

Comment:
My preference would be to avoid varying this output with `--verbose`.

I anticipate that rate limit exemptions and other forms of support need are a notable enough use case to always show Account URI. 

For Account Thumbprint, I'm happy to just cut it. While I personally find it useful, I realize its probably not a good decision for Certbot. Can always add it later.

What do you think?

Reply:
It would be *great* to drop the account thumbprint, as for some reason the Buypass API doesn't return the account object with JWK stuff, so for some reason the registration key is set to `None` and I can't find the key anywhere else for now..

Without the thumbprint I have no objection to remove the verbosity check.

Reply:
Sorry, I just realized that in RFC8555 it's actually called "Account URL". Let's correct that.